src/c++/library/http_client.cc

```diff
@@ -313,6 +313,13 @@ class HttpInferRequest : public InferRequest {
       const std::vector<const InferRequestedOutput*>& outputs,
       triton::common::TritonJson::Value* request_json);
 
+  Error ConvertBinaryInputsToJSON(
```

Comment:
Is this only a thing for HTTP and not for GRPC?

Reply:
Correct

Reply:
What prevents JSON format for GRPC?

Reply:
☝️

Reply:
gRPC does not use JSON. It uses protobuf for data transfer.